repo/packages/C/chronos/0/config.json

```diff
@@ -72,7 +72,7 @@
     "chronos/hostname": {
       "description": "The advertised hostname stored in ZooKeeper so another standby host can redirect to this elected leader (default = localhost)",
       "type": "string",
-      "default": "`curl http://169.254.169.254/latest/meta-data/public-hostname`"
+      "default": "$HOST"
```

Comment:
Actually, this is an env variable in Marathon so it should work consistently for different cloud platforms.